docs/BRP-LO GBA mapping.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page-with-side-nav
+title: BRP-LO GBA mapping
+---
+# BRP-LO GBA mapping op Haal Centraal Personen Bevragen API
+
+De transformatie van de met de Haal Centraal Personen Bevragen API op te halen gegevens naar het response bericht gebeurd in twee stappen.
+Eerst wordt vanuit de BRP getransformeerd naar de GBA variant van de Haal Centraal Personen Bevragen API en dat wordt weer getransformeerd naar de proxy variant van de Haal Centraal Personen Bevragen API.
+
+De eerste stap wordt beschreven in het CSV bestand '[BRP-LO GBA mapping - Personen gba.csv](BRP-LO GBA mapping - Personen proxy.csv)', de tweede stap in het CSV bestand '[BRP-LO GBA mapping - Personen proxy.csv](BRP-LO GBA mapping - Personen proxy.csv)'.
+
+Enkele kolommen in beide bestanden verdienen een toelichting.
+
+| Mapping | Kolomnaam | Toelichting |
+| --- | --- | --- |
+| Personen gba | leveren door RvIG | Geeft aan hoe het gegeven door de RvIG middels de GBA variant van de API aan de proxy applicatie geleverd wordt. |
```

Comment:
Beetje wollige zin?

Beschrijft in welke vorm het BRP gegeven geleverd wordt

Reply:
Aangepast.